habitat-baselines/habitat_baselines/rl/multi_agent/pop_play_wrappers.py

```diff
@@ -0,0 +1,257 @@
+from collections import defaultdict
+from typing import Dict, List
+
+import torch
+
+from habitat_baselines.common.storage import Storage
+from habitat_baselines.common.tensor_dict import TensorDict
+from habitat_baselines.rl.multi_agent.utils import (
+    add_agent_names,
+    filter_agent_names,
+    get_agent_name,
+)
+from habitat_baselines.rl.ppo.policy import Policy, PolicyActionData
+from habitat_baselines.rl.ppo.updater import Updater
+
+
+class MultiPolicy(Policy):
+    """
+    Wraps a set of policies. Splits inputs and concatenates outputs.
+    """
+
+    def __init__(self):
+        self._active_policies = []
+
+    def set_active(self, active_policies):
```

Comment:
policy, updater and storage all have this `set_active` which is not part of the inherited class. I would consider adding an interface for that.
Also, set_active takes as input a policy, which means the policies are tracked externally? Would it be simpler to hav set_active take an index as input  and let the multiPolicy track the policies?

Reply:
I think it is best to let the AgentAccessMgr track the policies. The wrappers are meant to be light. `set_active` is only used by these multi-agent wrappers and not single agent variants so I don't think it belongs in the base interfaces.